### PR TITLE
Convert helpers to Kotlin

### PR DESCRIPTION
**Pull request by jakub-gonet**
## Test plan

Tested on Example App.

### CODE REVIEW

File: android/lib/src/main/java/com/swmansion/gesturehandler/ViewConfigurationHelper.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.swmansion.gesturehandler
+
+import android.view.View
+import android.view.ViewGroup
+
+interface ViewConfigurationHelper {
+  fun getPointerEventsConfigForView(view: View): PointerEventsConfig?
+  fun getChildInDrawingOrderAtIndex(parent: ViewGroup, index: Int): View?
```
Comment (j-piasecki) [inline android/lib/src/main/java/com/swmansion/gesturehandler/ViewConfigurationHelper.kt:8]: In the implementation the return types are not nullable, shouldn't they also be there?
```suggestion
  fun getPointerEventsConfigForView(view: View): PointerEventsConfig
  fun getChildInDrawingOrderAtIndex(parent: ViewGroup, index: Int): View
```

File: android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootHelper.kt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      UiThreadUtil.runOnUiThread { tryCancelAllHandlers() }
+    }
+  }
+
+  companion object {
+    private const val MIN_ALPHA_FOR_TOUCH = 0.1f
+    private fun findRootViewTag(viewGroup: ViewGroup): ViewGroup {
+      UiThreadUtil.assertOnUiThread()
+      var parent: ViewParent? = viewGroup
+      while (parent != null && !(parent is ReactRootView || RNGHModalUtils.isDialogRootViewGroup(parent))) {
```
Comment (j-piasecki) [inline android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootHelper.kt:125]: Personally, I would change that expression, not sure if it is better though
```suggestion
      while (parent != null && parent !is ReactRootView && !RNGHModalUtils.isDialogRootViewGroup(parent)) {
```

Comment (jakub-gonet) [reply android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootHelper.kt]: Makes sense, fixed in 36b27e3.

File: android/lib/src/main/java/com/swmansion/gesturehandler/ViewConfigurationHelper.kt (hunk 1 above)
Comment (jakub-gonet) [reply android/lib/src/main/java/com/swmansion/gesturehandler/ViewConfigurationHelper.kt]: I checked usages and they're not expecting nulls, that's the right call.
Fixed in 75fba25.

**Review by j-piasecki — APPROVED** [review]

**Pull request merged**